.coveragerc

```diff
@@ -61,6 +61,7 @@ omit =
     homeassistant/components/asterisk_mbox/*
     homeassistant/components/aten_pe/*
     homeassistant/components/atome/*
+    homeassistant/components/automate/*
```

Comment:
This excludes the configuration flow from the test coverage, which is not wished for.
Please adjust this to only exclude the files you don't have tests for.